infra/modules/agentcore-iam-role/bedrock-agentcore-policy.tf

```diff
@@ -41,7 +45,9 @@ resource "aws_iam_policy" "ecr_permissions" {
         Action = [
           "ecr:GetAuthorizationToken"
         ]
-        Resource = "*"
+        Resource = [
+          "arn:aws:ecr:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:*"
+        ]
```

Comment:
As far as I can tell from the [ECR IAM reference](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonelasticcontainerregistry.html) this action doesn't actually support any resource constraint? But if this keeps the scanners happy then sure I guess...

Reply:
Resolved in b466748dc3150a89517a51cf221ab7fa150231d7